modal/_utils/time_utils.py

```diff
@@ -0,0 +1,14 @@
+# Copyright Modal Labs 2025
+from datetime import datetime
+
+
+def timestamp_to_local(ts: float, isotz: bool = True) -> str:
+    if ts > 0:
+        locale_tz = datetime.now().astimezone().tzinfo
+        dt = datetime.fromtimestamp(ts, tz=locale_tz)
+        if isotz:
+            return dt.isoformat(sep=" ", timespec="seconds")
+        else:
+            return f"{datetime.strftime(dt, '%Y-%m-%d %H:%M')} {locale_tz.tzname(dt)}"
+    else:
+        return None
```

Comment:
Should the return time be `Optional[str]`? (Or should we keep str but use `""` as the null return value?)